### ISSUE TITLE
Missing natural=beach?

### ISSUE THREAD

**Issue by nvkelso** [tilezen/vector-datasource#1575]
* **What did you see?** Nothing
* **What did you expect to see?** Beach polygon, with surface tag
* **What map location are you having problems with?** https://www.openstreetmap.org/way/575235542#map=17/51.07680/1.18310

These used to be on the map. When I view San Francisco they are missing from the map now.

**Comment by nvkelso** [comment]
QA followup for https://github.com/tilezen/vector-datasource/issues/1425.

**Comment by nvkelso** [comment]
Another example, which is odd since the beach is really sand at low tide?

https://www.openstreetmap.org/way/93759701#map=16/51.3880/1.3696

**Comment by zerebubuth** [comment]
I looked at the data in the tiles, and I see `kind: beach` polygons in both Margate and Folkestone with the appropriate `kind_detail` / `surface` values. There didn't seem to be anything funky about the polygons, so I poked around in the style file and found that commenting out [one line](https://github.com/tangrams/bubble-wrap/blob/4832b1c2128bf113f6b31ea1148a93a189a0520d/bubble-wrap-style.yaml#L5114) made the beaches appear again. Looks like there's a block which turns some kinds off and then selectively re-enables them in sub-blocks, except it never re-enables `kind: beach`?
